pkg/generic/thrift/json.go

```diff
@@ -97,6 +129,7 @@ func NewReadJSON(svc *descriptor.ServiceDescriptor, isClient bool) *ReadJSON {
 		svc:              svc,
 		isClient:         isClient,
 		binaryWithBase64: true,
+		useDynamicGo:     false,
```

Comment:
Why named useDynamicgo, previous definetion is dynamicgoEnabled

Reply:
I forgot to rename this. thank you

Reply:
fixed